docs/function-instructions.md

```diff
@@ -95,6 +99,7 @@ It returns an array of books and authors that represents all books _currently ch
 ```javascript
 booksInPossession(account, books, authors);
```

Comment:
To be consistent with how we are naming the other functions (i.e., starting with a verb), maybe we can call this function `getBooksPossessedByAccount()`?